charm/docs/tutorial/getting-started.md

```diff
@@ -43,26 +47,32 @@ storage, Redis or PostgreSQL.
 
 Set the allowed hosts. In this example every host is allowed. For a production environment
 only the used hosts should be allowed.
-```
+
+### Deploy the charms:
```

Comment:
Perhaps this heading should be updated to something like 
` ### Configure the NetBox charm`

Does this step need to happen before the other charms are deployed?

Reply:
I have put a new NetBox header that includes both things